### ISSUE TITLE
How to use npm run serve:build and deploy to production

### ISSUE THREAD

**Issue by a-barbieri** [ReactJSResources/react-webpack-babel#319]
Hi, I've just published the app I built with this starter kit on my remote server. Running `npm run serve:build` I'm able to see the application online. Very happy.

If I close the ssh connection the server still runs which is amazing. 

My questions are: 

- should I care about stopping it? 
- is it ok to run it multiple times?

**Comment by LeonardoGentile** [comment]
`serve:build` should be needed only when serving your app in local during development.
Simply running : `npm run build` should be fine, your app is compiled, minified etcr.

Then how you deploy should depend on your environment, you should configure a web server (apache, nginx or others) to serve your app that at this point should only be static files (I'm just talking about the frontend).

So, no you don't need serve:build and you clearly can (and should) stop it.

**Comment by a-barbieri** [comment]
Perfect. 
BTW I've found out that [this line](https://github.com/ReactJSResources/react-webpack-babel/blob/master/webpack.production.config.js#L19) was preventing secondary routes to render assets properly.

I changed this:
```javascript
// webpack.production.config.js
  output: {
    publicPath: './',
    //... some code
  },
```
to this (basically removing the dot from `publicPath`)
```javascript
// webpack.production.config.js
  output: {
    publicPath: '/',
    //... some code
  },
```
and everything works fine.

I wonder if it's correct. If it is I can PR it.
